WinUIGallery/ControlPages/ListViewPage.xaml

```diff
@@ -234,16 +234,16 @@ source code for the WinUI Gallery. -->
                 HorizontalAlignment="Left">
                     <ListView.ItemsPanel>
                         <ItemsPanelTemplate>
-                            <ItemsStackPanel x:Name="GroupedStackPanel" AreStickyGroupHeadersEnabled="False" Loaded="StackPanel_loaded" />
+                            <ItemsStackPanel x:Name="GroupedStackPanel" AreStickyGroupHeadersEnabled="False" Loaded="StackPanel_loaded"/>
                         </ItemsPanelTemplate>
                     </ListView.ItemsPanel>
 
                     <ListView.GroupStyle>
                         <GroupStyle >
                             <GroupStyle.HeaderTemplate>
                                 <DataTemplate x:DataType="local1:GroupInfoList">
-                                    <Border AutomationProperties.Name="{x:Bind Key}">
-                                        <TextBlock Text="{x:Bind Key}" Style="{ThemeResource TitleTextBlockStyle}"/>
+                                    <Border AutomationProperties.AccessibilityView="Raw">
```

Comment:
Please make the same changes in ControlPagesSampleCode text file so that the source will match.

Reply:
Totally forgot about, good catch!